chainbridge/src/lib.rs

```diff
@@ -284,20 +359,22 @@ impl<T: Trait> Module<T> {
         // Use default in the case it doesn't already exist
```

Comment:
Shouldn't we ensure here that who is a trusted relayer?

Reply:
This method is called from `acknowledge_proposal`, which ensures the caller is a relayer.

Reply:
clear now